src/install/aws-logs/aws_services/see-your-data.mdx

```diff
@@ -0,0 +1,37 @@
+---
+headingText: See your data
+componentType: default
+---
+Use your license key in New Relic and complete the AWS account integration.
```

Comment:
What does this mean?

Reply:
I wrote "Use your license key..." because in the first case, the user has to enter the key, but in the second case, the user just has to continue without any key related action (The key will be available there).

Therefore, I am removing the "License key" part from the sentence. The updated sentence is:

"To complete the AWS account integration in New Relic, follow these steps:"